localization/hapi-pseudonimization-interceptor/README.md

```diff
@@ -0,0 +1,326 @@
+# HAPI FHIR Pseudonymization Interceptor
+
+This interceptor provides automatic pseudonymization for BSN tokens in FHIR DocumentReference resources (`DocumentReference.subject.as(Identifier)`). It converts BSN tokens to pseudonyms for storage and back to audience-specific tokens for retrieval.
+
+## Overview
+
+The interceptor handles the conversion between:
+- **BSN Tokens** (transport format): `token-{audience}-{transformedBSN}-{nonce}`
+- **Pseudonyms** (storage format): `ps-{audience}-{transformedBSN}`
+
+All DocumentReference resources are stored with pseudonyms internally, but clients interact using BSN tokens.
+
+## Configuration
+
+The following environment variables can be configured:
+
+| Variable | Default | Description |
+|----------|---------|-------------|
+| `PSEUDO_BSN_SYSTEM` | `http://example.com/pseudoBSN` | System URL for pseudonyms (internal) |
+| `BSN_TOKEN_SYSTEM` | `http://example.com/BSNToken` | System URL for BSN tokens (client-facing) |
```

Comment:
Can you use;

`http://fhir.nl/fhir/NamingSystem/bsn-transport-token`
and
`http://fhir.nl/fhir/NamingSystem/bsn-pseudonym`

as code systems? That's what I used in the Knooppunt itself.